pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1115,10 +1167,12 @@ func ValidateVirtualMachineInstanceMandatoryFields(field *k8sfield.Path, spec *v
 	var causes []metav1.StatusCause
 
 	requests := spec.Domain.Resources.Requests.Memory().Value()
+	if requests != 0 {
+		return causes
```

Comment:
Might make sense to return `nil` instead to indicate "no errors". Had to look at this twice before I noticed requests should not be empty for the validation to succeed.

Reply:
I would prefer to let it as it is because if I return `nil` here, it would change what was returned in the old code. See https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go#L1119